release-notes/unreleased/2646-dont-hide-content-behind-keyboard.yml

```
@@ -0,0 +1,6 @@
+issue_key: 2646
+show_in_stores: true
```

:upside_down_face: since this is only a malte feature and only active in a  few regions, I'd set this to false
```suggestion
show_in_stores: false
```